iconiatech/jisort-api
# Implements the whatsapp webhook core functions

#### What does this PR do?
* Create the whatsapp webhook endpoint to respond to whatsapp messages
* Links whatsapp API to the core application

#### Description of Task to be completed?
Have the following endpoint working
1. `POST /webhook`

#### How should this be manually tested?
* After clonig the REPO, cd into it and run npm install
The run `npm run start` to start the application

* Using postman test all the endpoints above
To test protected endpoint, get the TOKEN from the login endpoint
key: Authorization value: Bearer

* Test the company whatsapp number and ensure the endpoint responds
